spec/linter-markdown-spec.js

```diff
@@ -30,14 +30,11 @@ describe('The remark-lint provider for Linter', () => {
 
     it('verifies the first message', async () => {
       const messages = await lint.provideLinter().lint(editor);
-      expect(messages[0].type).toEqual('Warning');
-      expect(messages[0].text).not.toBeDefined();
-      expect(messages[0].html).toEqual(
-        '<span class="badge badge-flexible">remark-lint:' +
-        'no-unused-definitions</span> Found unused definition'
-      );
-      expect(messages[0].filePath).toMatch(/.+definition-use-invalid\.md$/);
-      expect(messages[0].range).toEqual([[2, 0], [2, 58]]);
+      expect(messages[0].severity).toEqual('warning');
+      expect(messages[0].location.file).toMatch(/.+definition-use-invalid\.md$/);
```

Comment:
Same thing, `toBe` instead of `toEqual` (assuming `remark` outputs OS specific paths).